workflow/scripts/finalize_mhcII_input.py

```
@@ -63,7 +64,7 @@ def main():
             print(f"Single-end fastq file detected. Splitting into two files...")
             print(f"Forward reads: {out_fwd}")
             print(f"Reverse reads: {out_rev}")
-            in_fh = open(input_files[0], "r")
+            in_fh = open(inputfile[0], "r")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)